packages/shared/lib/integrations/scripts/connection/github-app-oauth-post-connection.ts

```diff
@@ -1,19 +1,21 @@
 import type { InternalNango as Nango } from './connection.manager.js';
 import { AuthModes, OAuth2Credentials } from '../../../models/Auth.js';
+import axios from 'axios';
 
 export default async function execute(nango: Nango) {
+    const connection = await nango.getConnection();
```

Comment:
move connection to InternalNango

Reply:
InternalNango is actually an interface with no attributes, just functions to fetch Nango internal data. I am not sure I want to start making it a object with a state.